routers/web/repo/commit.go

```diff
@@ -252,7 +252,7 @@ func FileHistory(ctx *context.Context) {
 func Diff(ctx *context.Context) {
 	ctx.Data["PageIsDiff"] = true
 	ctx.Data["RequireHighlightJS"] = true
-	ctx.Data["RequireSimpleMDE"] = true
+	ctx.Data["RequireEasyMDE"] = true
```

Comment:
Can we do away with this variable and just detect the presence of certain DOM elements in JS instead?

Reply:
Good idea!

Maybe async loading should work, or we could get this merged if it isn't worse than before 😊

Reply:
@silverwind 

Could we move on? I think the refactoring of `RequireEasyMDE` can be done in a separate PR. This PR has done enough. Removing `RequireEasyMDE` might involve unrelated changes.